src/planscape/forsys/forsys_request_params.py

```diff
@@ -365,12 +365,12 @@ def __init__(self, request: HttpRequest) -> None:
 
         self.db_params = DbRequestParamsForGenerationFromDb(request)
         self._validate_scenario(self.db_params.scenario)
-        
+
         self._read_db_params()
 
     def _validate_scenario(self, scenario: Scenario):
         status = scenario.status
-        if not (status == Scenario.ScenarioStatus.INITIALIZED or
+        if not (status == Scenario.ScenarioStatus.INITIALIZED or status == Scenario.ScenarioStatus.PROCESSING or
```

Comment:
Can you please update the Exception string, too?

Reply:
Also: what's the reasoning behind making processing ok? I originally wanted to prevent kicking off a lambda run if the status of a scenario is processing (since that implies a lambda run is already running).

Reply:
I see!

Yea: can you do that instead of changing the logic _validate_scenario? I think It'd be easier if code changes associated with the prototype are concentrated in as few functions as possible.

Reply:
reverted this change